drivers/tm-fuse/lfs.c

```diff
@@ -172,12 +180,18 @@ int lfs_modal_lza2map_addr(
 
 	case MODE_FAME:
 	case MODE_FAME_DESC:	// from lfs_shadow.py::shadow_offset()
-		ig_book_num = lza >> 33;	// 20-bit combo
-		ig = ig_book_num >> 13;		// top 7 bits, final
-		ig_book_num &= 8191;		// 13-bit mask, final
-		*map_addr = G.shadow_igstart[ig] +
-			    (ig_book_num * G.book_size) +
-			    shelf_book_offset;
+		if (G.bii_mode == MODE_PHYSADDR) {
+			/* BII.MODE_PHYSADDR */
+			*map_addr = lza;
```

Comment:
Rename the variable "lza" to "bookID" to match the Python per yesterday's discussion.